packages/teleterm/src/services/grpcCredentials/grpcClientCredentials.ts

```diff
@@ -0,0 +1,59 @@
+import { ChannelCredentials, credentials } from '@grpc/grpc-js';
+import { RuntimeSettings } from 'teleterm/mainProcess/types';
+import {
+  readGrpcCert,
+  generateAndSaveGrpcCert,
+  shouldEncryptConnection,
+} from './helpers';
+import { GrpcCertName } from './types';
+
+export async function getGrpcClientCredentials(
```

Comment:
What do you think about dropping `grpc` from function and file names? They're already living in the `grpcCredentials` directory anyway.

Reply:
Ah yes, look much better now